.travis.yml

```diff
@@ -1,13 +1,13 @@
 dist: xenial
-sudo: required
+
 language: java
 
 jdk:
   - openjdk11
+  - openjdk12
```

Comment:
not sure about this, the build already takes around 11 minutes and running another JDK will double that time, won't it?

Reply:
If you build this in the "eclipse" organization, then most likely yes. If you do this in your personal account, then this will, most likely, run in parallel.

Reply:
Since we run it on Travis in order to integrate properly with the GitHub build checks, I do not think that we should add another variant. It is already slow enough ...

Reply:
However we would gain some insights on upcoming issues.